tests/test_flow/test_persistence.py

```diff
@@ -175,8 +200,8 @@ def f(x, y):
 
     builder.delete("f")
 
-    @builder  # noqa: F811
-    def f(x, y):
+    @builder
+    def f(x, y):  # noqa: F811
```

Comment:
Hmm, weird. Flake8 seemed happy with the original way for me.

Reply:
When I run `flake8` from terminal (v `3.8.1`), it wants the original style too. But `pre-commit` `flake8` prefers the new style (v `3.7.9`). 

Looks like the behavior changed with `flake8` versions. I'll try updating the pre-commit flake8 version.

Reply:
Looks like the behavior changed for python version instead of flake8 version. I'm just going to add it on both lines since we support python 3.6+. 

https://gitlab.com/pycqa/flake8/-/issues/583

Reply:
Wow, that sucks! Oh well.